modules/ROOT/pages/security-architecture.adoc

```diff
@@ -0,0 +1,135 @@
+= Anypoint Runtime Fabric Security Architecture
+
+The Runtime Fabric architecture model provides a secure way to manage MuleSoft applications and API gateways deployed onto Runtime Fabrics anywhere, while providing a data flow capability that ensures you own, and controls, the source and destination of all your data.
```

Comment:
```suggestion
The Runtime Fabric architecture model provides a secure way to manage MuleSoft applications and API gateways deployed onto Runtime Fabrics anywhere, while providing a data flow capability that ensures you own and control the source and destination of all your data.
```

Reply:
Can you shorten this paragraph or break it up?  Do you need to say "providing a data flow capability?

The Anypoint Runtime Fabric architecture model provides a secure way to manage MuleSoft applications and API gateways deployed onto Runtime Fabrics anywhere; while ensuring that you own and control the source and destination of all of your data.

Reply:
First mention in text: Anypoint Runtime Fabric
Global - Next mention: Runtime Fabric